gps/result_test.go

```diff
@@ -104,6 +110,8 @@ func testWriteDepTree(t *testing.T) {
 	if _, err = os.Stat(filepath.Join(tmp, "bitbucket.org", "sdboyer", "withbm")); err != nil {
 		t.Errorf("Directory for bitbucket.org/sdboyer/withbm does not exist")
 	}
+
+	wg.Wait()
```

Comment:
Only the logging goroutines were causing actual race failures. I fixed this one the easy way when I started chasing these down, but I will revert it.